env/includes/settings/tk-shotgun.yml

```diff
@@ -48,6 +54,9 @@ settings.tk-shotgun.asset_step:
 # project
 settings.tk-shotgun.project:
   apps:
+    tk-multi-createdelivery2: "@settings.tk-multi-createdelivery2.project"
+    tk-multi-importmedia: "@settings.tk-multi-importmedia"
+    tk-multi-ingestdelivery: "@settings.tk-multi-ingestdelivery"
```

Comment:
Ingest is missing from the shotgun project env our config but seems like it makes sense here.